src/tests/components/__snapshots__/LoginForm.test.js.snap

```diff
@@ -104,7 +104,12 @@ ShallowWrapper {
             >
               New to My Diary ?
               <span>
-                Sign up
+                <Link
+                  replace={false}
```

Comment:
@kakaemma Consider adding the snapshots to the `gitignore` file. They are unnecessary on remote repository

Reply:
@samuelbwambale, according to the jest documentation https://jestjs.io/docs/en/snapshot-testing#should-snapshot-files-be-committed, snapshots should also be commited